PORT-4056-docs-for-relative-file-path-in-gitops (#390)
port-labs/port-docs: docs/build-your-software-catalog/sync-data-to-catalog/git/_file_properties_template.md

> The content of files in the repository can be pushed to string properties of entities with a simple reference.

Reviewer: ```suggestion
It is possible to use the contents of files in the repository as the value for entity properties using a simple reference.
```

> The content of files in the repository can be pushed to string properties of entities with a simple reference.

Reviewer: Note about adding all of this content as a separate template file: because this file is imported as an external source, the subheadings you created (`####`) will not appear in the TOC on the right side of the page. I would suggest limiting the templates to code blocks if you think they will be used in other places

The following example will read the string content of `~/module1/README.md` and upload it to `myStringProp`.

> Repository folder structure:

Reviewer: ```suggestion
Repository folder structure used for the example:
```

```
root
|
+- port.yml
|
+-+ module1
|   |
|   +- README.md
|   |
|   +-+ src
...
```

port.yml:

```yaml showLineNumbers

---

> properties:

Reviewer: Extend this `port.yml` spec example to look a bit more like a complete entity (doesn't have to be full, but at least show things such as the identifier so the reader has an easier time understanding the context).

Apply the same fix to the 2nd code example in the relative paths section

  myStringProp: file://module1/README.md
```

> #### Relative paths

Reviewer: ```suggestion
#### Using relative paths
```

> Relative paths are also supported, relative to the port.yml spec file, so `file://./` would mean the same route as `port.yml` and `file://../` would mean one directory above it.

Reviewer: ```suggestion
It is also possible to use paths relative to the location of the `port.yml` spec file.

For example: `file://./` is used to reference a file in the same directory as the `port.yml` file. `file://../` is used to reference a file that is one directory above and so on.
```

The following example reads `README.md` and `module1/requirements.txt` using paths relative to `port.yml`

> Repository folder structure:

Reviewer: ```suggestion
Repository folder structure used for the example:
```
